src/server/index.jsx

```diff
@@ -104,14 +104,20 @@ server
 
       const headHelmet = Helmet.renderStatic();
 
-      const styleTags = sheet.getStyleElement();
+      let inlineCss = '';
+      try {
+        // eslint-disable-next-line prefer-destructuring
+        inlineCss = renderToString(sheet.getStyleElement()).match(
+          /^<style (?:[^>]+)>([\S\s.]+)<\/style>$/,
+        )[1];
+      } catch (e) {} // eslint-disable-line no-empty
```

Comment:
_Could_ revert back to the original method, if we wanted to avoid a regex:

```js
      let inlineCss = '';
      // `getStyleElement()` doesn't always return an array, so wrap in a try-catch.
      // TODO: fix in https://github.com/BBC-News/simorgh/issues/976
      try {
        inlineCss = sheet.getStyleElement().reduce(
          (inlineStyles, currentStylesheet) =>
            currentStylesheet
              ? `${inlineStyles}${
                  // eslint-disable-next-line no-underscore-dangle
                  currentStylesheet.props.dangerouslySetInnerHTML.__html
                }`
              : inlineStyles,
          '',
        );
      } catch (e) {} // eslint-disable-line no-empty
```

...but the disadvantage of that approach is that if `sheet.getStyleElement()` returns a single object instead of an array, we lose the styles of that object (unless we modify the above code even further).

The advantage of the Regex method is that whatever CSS gets rendered, gets captured.

The _disadvantage_ of the Regex method is that it's brittle: if the `renderToString` or internal `render()` method of the Styled Components library changes, there's a risk we'll deliver a page with no styles.